app/View/Cerebrates/add.ctp

```diff
@@ -16,10 +17,10 @@ $fields = [
     [
         'field' => 'org_id',
         'label' => 'Owner Organisation',
-        'type' => 'select',
         'options' => $dropdownData['org_id'],
         'class' => 'span6',
-        'searchable' => 1
+        'searchable' => 1,
```

Comment:
Not sure if we should keep it. Seems to be a useless option (if it is, remove subsequent usage)